readthedocs/proxito/views/serve.py

```diff
@@ -477,8 +477,15 @@ def get(self, request):
         project, we serve it directly.
         """
         project = request.unresolved_domain.project
+        # Check if project is delisted
```

Comment:
Not really sure this comment, or the one below it are adding value, since they're basically just saying the same thing as the line of code.

I think we probably just want to note this functionality in the docstring, instead of comments?